pkgs/tools/networking/openconnect_pa/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, pkgconfig, vpnc, openssl ? null, gnutls ? null, gmp, libxml2, stoken, zlib, autoreconfHook } :
+
+assert (openssl != null) == (gnutls == null);
+
+let
+  version = "7.08";
```

Comment:
When I see this I'm assuming that this is going to be the `7.08` release of `openconnect_pa`.

However the `rev` used for the fetch is `"e5fe063a087385c5b157ad7a9a3fa874181f6e3b"` which isn't a release but a [commit ](https://github.com/dlenski/openconnect/commit/e5fe063a087385c5b157ad7a9a3fa874181f6e3b) from the repo.

So this should be in the format `unstable-YYYY-MM-DD`.

##### See also [`Packaging Naming`](https://nixos.org/nixpkgs/manual/#sec-package-naming) in the nixpkgs docs.
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23".